pkg/segment/segment.go

```diff
@@ -0,0 +1,163 @@
+package segment
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/openshift/odo/pkg/preference"
+	"github.com/openshift/odo/pkg/version"
+	"github.com/pborman/uuid"
+	"github.com/pkg/errors"
+	"golang.org/x/term"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+// Writekey will be the API key used to send data to the correct source on Segment
+var WriteKey = "R1Z79HadJIrphLoeONZy5uqOjusljSwN"
+
+type Client struct {
+	// SegmentClient helps interact with the segment API
+	SegmentClient analytics.Client
+	// Preference points to the global odo config
+	Preference *preference.PreferenceInfo
+	// TelemetryFilePath points to the file containing anonymousID used for tracking users
```

Comment:
```suggestion
	// TelemetryFilePath points to the file containing anonymousID used for tracking odo commands executed by the user
```
Nitpick. But this changes the meaning to a good extent. If someone judges odo's compliance by reading this comment, it would be a bad/incorrect impression that they would gather.